### PR TITLE
luxon 1.27.0

### PR DESCRIPTION
**Pull request by Eveliyse**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/moment/luxon/blob/master/CHANGELOG.md#1270-2021-05-08
https://github.com/moment/luxon/pull/926
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@Eveliyse thx!

File: types/luxon/src/datetime.d.ts [hunk 1]
```diff
@@ -32,6 +32,8 @@ export interface ToRelativeOptions extends ToRelativeCalendarOptions {
      * @default 0
      */
     padding?: number;
+    /** If omitted, the method will pick the unit. */
```
Comment (CarsonF) [inline types/luxon/src/datetime.d.ts:35]: The method also picks the best unit when this is given too. How about:
```suggestion
    /** A subset of units that the method will pick the best one from */
```

Comment (Eveliyse) [reply types/luxon/src/datetime.d.ts:35]: Should the comment better reflect all 3 behaviors?

```
A single unit or an array of units. If an array is supplied, the method will pick the best one to use from the array. If omitted, the method will pick the unit from a default set.
```

Optionally is it of any advantage to try and reuse wording from the actual package?
https://github.com/moment/luxon/pull/926/files#diff-5146f40a8c8111b14e5ab3407eff80c7a6d2f3c92a4bf4ccd4647bff32d47e38R1850

Comment (CarsonF) [reply types/luxon/src/datetime.d.ts:35]: Yeah that's good with me too!

**Review by CarsonF — APPROVED** [review]
Thanks @Eveliyse!

**Review by peterblazejewicz — APPROVED** [review]

**Comment by Eveliyse** [comment]
Ready to merge

**Pull request merged**